scripts/e2etests/VSUtils.ps1

```diff
@@ -169,6 +169,45 @@ function UninstallVSIX
     return $true
 }
 
+function DowngradeVSIX
+{
+    param(
+        [Parameter(Mandatory=$true)]
+        [string]$vsixID,
+        [Parameter(Mandatory=$true)]
+        [ValidateSet("15.0")]
+        [string]$VSVersion,
+        [Parameter(Mandatory=$true)]
+        [int]$VSIXInstallerWaitTimeInSecs
+    )
+
+    $VSIXInstallerPath = GetVSIXInstallerPath $VSVersion
+
+    Write-Host 'Downgrading VSIX...'
+    $p = start-process "$VSIXInstallerPath" -Wait -PassThru -NoNewWindow -ArgumentList "/q /a /d:$vsixID"
+
+    if ($p.ExitCode -ne 0)
+    {
+        if($p.ExitCode -eq 1002)
```

Comment:
Yup, I wasn't sure about that, so that's why I already tagged @rohit21agrawal in the original  PR message to validate it.